src/CosmosDB/CosmosDB/help/New-AzCosmosDBAccount.md

```diff
@@ -291,6 +291,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PublicNetworkAccess
```

Comment:
Did you generate these using Update-MarkdownHelp cmdlet ? If not, please do so, that might fix the build error in the CI.

Reply:
Gotcha Thanks